internal/dag/httpproxy_processor.go

```diff
@@ -137,7 +160,17 @@ func (p *HTTPProxyProcessor) computeHTTPProxy(proxy *projcontour.HTTPProxy) {
 
 			// Check if FallbackCertificate && ClientValidation are both enabled in the same vhost
 			if tls.EnableFallbackCertificate && tls.ClientValidation != nil {
-				sw.SetInvalid("Spec.Virtualhost.TLS fallback & client validation are incompatible together")
+				sw.SetInvalid("Spec.Virtualhost.TLS fallback & client validation are incompatible")
+				return
+			}
+
+			// Fallback certificates and authorization are
+			// incompatible because fallback installs the routes on
+			// a separate HTTPConnectionManager. We can't have the
+			// same routes installed on multiple managers with
+			// inconsistent authorization settings.
+			if tls.EnableFallbackCertificate && proxy.Spec.VirtualHost.AuthorizationConfigured() {
+				sw.SetInvalid("Spec.Virtualhost.TLS fallback & client authorization are incompatible")
```

Comment:
it'd be nice to add a test case to `status_test.go` for this.

Reply:
This is tested in `authorization_test.go`; do we need to double-test?

Reply:
It seems consistent with the existing testing approach to have a test case in `status_test.go`. Why deviate from that pattern now?

Reply:
I added a test; but I do hate `status_test.go` a lot and we should kill it with fire :)